src/config.js

```diff
@@ -63,6 +67,20 @@ function debug () {
   if (HEROKU_DEBUG) return parseInt(HEROKU_DEBUG)
   return 0
 }
+function skipAnalytics (config) {
+  if (process.env['TESTING'] && process.env['TESTING'] === '1') {
+    return true
+  } else if (config.userConfig.skipAnalytics) {
+    return true
+  } else {
+    return false
+  }
+}
+
+let loadUserConfig = function (configDir) {
+  const rawdata = fs.readFileSync(`${configDir}/user_config.json`, 'utf8')
```

Comment:
@KarateCowboy Can be shortened with [fs.readJsonSync](https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md)